documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -113,6 +113,13 @@ VMware only: You must have the minimal set of xref:vmware-privileges_{context}[V
 VMware only: You must create a xref:creating-vddk-image_{context}[VMware Virtual Disk Development Kit (VDDK)] image.
```

Comment:
@RichardHoch unrelated to this PR - this needs to change as VDDK is optional in 2.5